src/screens/HomeScreen.tsx

```diff
@@ -10,24 +10,38 @@ import { i18n } from "locale";
 interface Props {}
 
 const HomeScreen: FC<Props> = () => {
+  const [inputValue1, setInputValue1] = useState<string>("");
+  const [inputValue2, setInputValue2] = useState<string>("");
   return (
     <ScreenWrapper>
       <div className={css(styles.homeScreen)}>
         <h3>{i18n.t("header:title")}</h3>
-        <StyledInput value="" placeholder="Search..." />
-        <StyledInput value="" placeholder="Error..." error="Error Text!!!" />
+        <StyledInput
+          value={inputValue1}
+          onChange={(event) => setInputValue1(event.target.value)}
```

Comment:
ok, but u should pass a callback to onChange, so create a function above return , for example: 
`const handleInputValue = (value) => {  setInputValue1(value) }`
and pass it like that: onChange={handleInputValue}